openedx/core/djangoapps/catalog/management/commands/sync_course_runs.py

```diff
@@ -0,0 +1,55 @@
+"""
+Sync course runs from catalog service.
+"""
+import logging
+
+from django.core.management.base import BaseCommand
+from opaque_keys.edx.keys import CourseKey
+
+from catalog.utils import get_catalog_course_runs
+from content.course_overviews.models import CourseOverview
+
+logger = logging.getLogger(__name__)
+
+
+def fetch_catalog_course_runs():
```

Comment:
I typically see functions like these implemented as methods on the `Command` class. If they turn out to be more general purpose, we can break them out to a utility module later.